services/QuillLMS/app/helpers/content_hubs_helper.rb

```diff
@@ -0,0 +1,295 @@
+      },
+      {
+        display_name: "Global Conflict (1914-1945 CE)",
+        description: "From 1914 to 1945, the world endured two major conflicts ---World War I and World War II---which resulted in immense human suffering, advancements in military technology, and major shifts in international relations. The interwar period began with efforts to prevent further conflict, but this work was hindered by economic instability and the rise of totalitarian regimes.",
+        unit_template_id: 591,
+        oer_unit_website: "https://www.oerproject.com/1200-to-the-Present/Unit-7",
+        oer_unit_teacher_guide: "https://www.oerproject.com/OER-Materials/OER-Media/PDFs/1200/Unit7/Unit-7-Guide",
+        all_oer_articles: "https://docs.google.com/document/d/1nO78eUYRNORQQwnWy9jUmwTbxmk1dNNd/edit?usp=drive_link&ouid=110057766825806701001&rtpof=true&sd=true",
+        all_quill_articles_href: 'https://docs.google.com/document/d/14mnkQ75WILd6WsGch6AfG4XxKaiY_u8LHR9ZQyg2DD8/edit?usp=sharing',
+        quill_teacher_guide_href: '',
+        activities: global_conflict_activities
+      },
+      {
+        display_name: "Cold War and Decolonization (1945-1990 CE)",
+        description: "The second half of the 20th century was shaped by the dynamics of the Cold War and decolonization. The capitalist United States and communist Soviet Union rose as dominant powers, competing for global influence. Simultaneously, decolonization movements used a variety of methods, including both non-violent resistance and armed conflict, to achieve independence.",
+        unit_template_id: 598,
+        oer_unit_website: "https://www.oerproject.com/1200-to-the-Present/Unit-8",
+        oer_unit_teacher_guide: "https://www.oerproject.com/OER-Materials/OER-Media/PDFs/1200/Unit8/Unit-8-Guide",
+        all_oer_articles: "https://docs.google.com/document/d/1tHmlCgf8FgIoS0Ipdvj2Zuw3AKDYPU14/edit?usp=drive_link&ouid=110057766825806701001&rtpof=true&sd=true",
+        all_quill_articles_href: 'https://docs.google.com/document/d/1Cg7ShOIbYoMa3NYUSh_l2Fx5B4CiZODEW1_vSVTagCw/edit?usp=sharing',
+        quill_teacher_guide_href: '',
+        activities: cold_war_and_decolonization_activities
+      },
+      {
+        display_name: "Globalization (1990-Present)",
+        description: "Coming Spring 2025!",
+        unit_template_id: nil,
+        oer_unit_website: "https://www.oerproject.com/1200-to-the-Present/Unit-9",
+        oer_unit_teacher_guide: "https://www.oerproject.com/OER-Materials/OER-Media/PDFs/1200/Unit9/Unit-9-Guide",
+        all_oer_articles: "https://docs.google.com/document/d/1UN4K3X6LgfhhU8ET0tagon92pYxMebAL/edit?usp=drive_link&ouid=110057766825806701001&rtpof=true&sd=true",
+        all_quill_articles_href: '',
+        quill_teacher_guide_href: '',
+        activities: globalization_activities
+      }
+    ]
+  end
+
+  def global_tapestry_activities
+    [
+      {
+        activity_id: 2750,
+        display_name: 'Wokou Pirates',
```

Comment:
This is a higher level issue, but do we really want to be hard-coding these display names?  This will mean that they don't match the actual activity names, and thus may result in discrepancies in Teacher reporting.  Maybe that's fine, and like I said, it's probably a question beyond the scope of this code, but I notice, for instance, that this activity is actually named "How did the Wokou Pirates Disrupt Sea Routes in East Asia?" in the database.

Reply:
Good question. I'll double check with Jamie. I think Wokou Pirates is actually just a placeholder here (the description also isn't ready yet for the curriculum team). A lot of the other activities have names that include things like "[Beta - Early Access]", which is part of why I hardcoded, but maybe it would be better to just go ahead and strip that string rather than actually hardcoding a name for each one. I do kind of like that it's easier to see which hash belongs to which activity for the developer, but that doesn't trump the discrepancy possibility you pointed out.